snuba/admin/views.py

```diff
@@ -47,12 +48,19 @@ def handle_invalid_json(exception: UnauthorizedException) -> Response:
     )
 
 
+@application.before_request
+def set_logging_context() -> None:
+    clear_contextvars()
```

Comment:
Just read up on this package, this is very cool. Something we might want to leverage for the api as well.

Reply:
yeah, i think once you start "thinking" in terms of structured logging it's an extremely useful tool. IMO it should be the basis for all logs, but admin is a safe place to start